test/files/neg/t7259.check

```diff
@@ -4,4 +4,7 @@ t7259.scala:1: error: not found: type xxxxx
 t7259.scala:8: error: type xxxxx is not a member of package annotation
 @annotation.xxxxx // error: not found: type scala
             ^
-two errors found
+t7259.scala:8: error: <none> does not extend scala.annotation.Annotation
```

Comment:
This additional messaging does not look right.

Reply:
So I found one way to solve this but I'm not sure if it's right